js/src/wp-seo-tinymce.js

```diff
@@ -166,16 +166,28 @@ var termsTmceId = "description";
 		}
 	}
 
+	/**
+	 * If #wp-content-wrap has the 'html-active' class, text view is enabled in WordPress.
+	 * TMCE is not available, the text cannot be marked and so the marker buttons are disabled.
+	 *
+	 * @returns {boolean} Whether the text view is active.
+	 */
+	function isTextViewActive() {
+		const contentWrapElement = document.getElementById( "wp-content-wrap" );
+		if ( ! contentWrapElement ) {
+			return false;
+		}
+		return document.getElementById( "wp-content-wrap" ).classList.contains( "html-active" );
+	}
+
 	/**
 	 * Check if the TinyMCE editor is created in the DOM. If it doesn't exist yet an on create event created.
 	 * This enables the marker buttons, when TinyMCE is created.
 	 *
 	 * @returns {void}
 	 */
 	function wpTextViewOnInitCheck() {
-		// If #wp-content-wrap has the 'html-active' class, text view is enabled in WordPress.
-		// TMCE is not available, the text cannot be marked and so the marker buttons are disabled.
-		if ( jQuery( "#wp-content-wrap" ).hasClass( "html-active" ) ) {
+		if ( isTextViewActive() ) {
```

Comment:
You should use the defensive way, just to an early return when `isTextViewActive` results in false